app/models/post.rb

```diff
@@ -27,4 +29,13 @@ def add_url_protocol(url)
       return "http://#{url}" unless url[/^https?:\/\//] || url[/^http?:\/\//]
       url
     end
+
+    def new_post_notification
+      @recipients = User.where("post_notification = ? AND email != ?", true, self.user.email)
```

Comment:
This looks good.  We might want to optimize this in the future, but I'll make a new ticket for this.